chalicelib_fourfront/checks/wfr_checks.py

```diff
@@ -1254,6 +1254,61 @@ def plac_seq_start(connection, **kwargs):
     action = wfr_utils.start_tasks(missing_runs, patch_meta, action, my_auth, my_env, fs_env, start, move_to_pc=True)
     return action
 
+@check_function(lab_title=None, start_date=None)
```

Comment:
I do not understand why this diff is showing. This file is the same as what's in the master branch.